launchable/test_runners/playwright.py

```diff
@@ -49,3 +61,85 @@ def subset(client):
 
 
 split_subset = launchable.CommonSplitSubsetImpls(__name__).split_subset()
+
+
+class JSONReportParser:
+    def __init__(self, client):
+        self.client = client
+
+    def parse_func(self, report_file: str) -> Generator[CaseEvent, None, None]:
+        data: Dict[str, Dict]
```

Comment:
Can we write example JSON file as a comment like https://github.com/launchableinc/cli/blob/main/launchable/test_runners/cucumber.py#L75?

Reply:
https://github.com/launchableinc/cli/pull/835/commits/033c6b5ae915fc6cc386f2c8aeb066bc062f1559